jsonrpcservice/__init__.py

```diff
@@ -530,7 +526,15 @@ def call_method(self, request, method_name, args, kwargs):
         except TypeError as e:
             raise InvalidParams(str(e))
 
-        response = method(*args, **kwargs)
+        try:
+            response = method(*args, **kwargs)
+        except Exception as e:
```

Comment:
Není ten reraise zbytečný? Mám za to že při syntax:
`try:
   raise ValueError('A very specific bad thing happened.') 
  finally:
    do_something()` se to chová stejně. Každopádně není to žádný breaker a ani možná nemám pravdu. :D